### PR TITLE
Creative Live! Camera Mount

### PR DESCRIPTION
**Pull request by DanKeeling**
### Which mods are added by this PR?
Trident, 2.4, any with 20x20 extrusions with space

### How was it tested?
Printed in 3 different orientations in ABS, found the best one and changed STL accordingly.  Included in instructions with print settings

### Any background context you want to provide?
I could not find a mount for the inexpensive web camera, so designed one in OnShape.

### Screenshots (if appropriate)
Included in the repo.
292704824-28ee8aaa-a486-46c3-86b7-506ef56032f8

### ⚠️⚠️**PLEASE READ AND COMPLETE THE CHECKLIST BELOW**⚠️⚠️

  * [x] I have read the rules available [here](https://github.com/VoronDesign/VoronUsers/wiki/Mod-Submission-Rules) and
        my mod adheres to these rules.
  * [x] This mod was created by myself and I agree to publish it under the repository
        [license](https://github.com/VoronDesign/VoronUsers/blob/master/LICENSE.md)

### CODE REVIEW

File: printer_mods/dankeeling/CreativeLiveCameraMount/README.md
Comment (chirpy2605) [inline printer_mods/dankeeling/CreativeLiveCameraMount/README.md:1]: Hi,

Could you make a few changes for me to help this through the process:

1. Make the image references in the README.MD relative to the files uploaded in this PR, i.e. Images/file.jpg
2. Remove the images not used in the README.MD file, i.e.:
Images/20231224_211032~2.jpg
Images/20231224_211032~3.jpg

File: printer_mods/dankeeling/CreativeLiveCameraMount/STL/Creative_Live_Camera.stl
Comment (chirpy2605) [inline printer_mods/dankeeling/CreativeLiveCameraMount/STL/Creative_Live_Camera.stl:1]: To help with printability of the part could you please add 0.4mm chamfers to the bottom and top of the part as it is orientated here. In particular the holes on the top and the bottom should have the same chamfers to help with pushing the screw through the mount.

The reason for the chamfers, particularly on the bottom of the print is to counter any elephants foot a user might print the part with, especially inside screw holes

File: printer_mods/dankeeling/CreativeLiveCameraMount/README.md
Comment (DanKeeling) [reply printer_mods/dankeeling/CreativeLiveCameraMount/README.md:1]: Adjusted Readme and removed extra images.

File: printer_mods/dankeeling/CreativeLiveCameraMount/STL/Creative_Live_Camera.stl
Comment (DanKeeling) [reply printer_mods/dankeeling/CreativeLiveCameraMount/STL/Creative_Live_Camera.stl:1]: Added .4mm chamfers as requested, good catch.

File: printer_mods/dankeeling/CreativeLiveCameraMount/CAD/Creative_Live_Camera.3mf
Comment (MapleLeafMakers) [inline printer_mods/dankeeling/CreativeLiveCameraMount/CAD/Creative_Live_Camera.3mf:1]: We don't allow 3mf files in the repository.  They're not useful as a CAD format, and slicer compatibility is too hit-or-miss, with no real benefit.

Comment (DanKeeling) [reply printer_mods/dankeeling/CreativeLiveCameraMount/CAD/Creative_Live_Camera.3mf:1]: Oh ok. Thanks for the help, removed from the branch.

**Comment by chirpy2605** [comment]
Aha! We're missing a meta file that contains the information to merge the modification. If you have a look here https://github.com/VoronDesign/VoronUsers/wiki/How-to-build-a-metadata.yml-file you'll hopefully see what needs to go into that file

**Comment by DanKeeling** [comment]
> Aha! We're missing a meta file that contains the information to merge the modification. If you have a look here https://github.com/VoronDesign/VoronUsers/wiki/How-to-build-a-metadata.yml-file you'll hopefully see what needs to go into that file

I forgot to include that, it's been added in the branch now.

**Comment by chirpy2605** [comment]
Looks like we're almost there! Thank you for sticking with it, we're learning a new system at the moment. One last thing that the automated test missed:

Can you rename Images/20231224_211032~2.jpg to not have the tilda, perhaps just use another underscore so that it conforms to:

> All filenames and foldernames in VoronUsers must not have spaces in them! Please stick to letters a-zA-Z, numbers 0-9, underscores _, hyphens -, brackets []() and periods .

You will also need to update the link for it in the .metadata.yml and the README.MD file as well.

**Comment by DanKeeling** [comment]
All good, new to this myself.  Image has been renamed and references in Readme and metadata are updated to reflect the new name.

File: printer_mods/dankeeling/CreativeLiveCameraMount/STL/Creative_Live_Camera.stl
Comment (chirpy2605) [inline printer_mods/dankeeling/CreativeLiveCameraMount/STL/Creative_Live_Camera.stl:1]: I think this file should be removed as the other STL is the one you modified.

File: printer_mods/dankeeling/CreativeLiveCameraMount/README.md
Comment (chirpy2605) [inline printer_mods/dankeeling/CreativeLiveCameraMount/README.md:1]: Could you please change the Amazon URL to remove the tracking data, it should be fine as:

https://www.amazon.com/Creative-Wide-Angle-Cancellation-Improved-Universal/dp/B0919XGLXH

Comment (DanKeeling) [reply printer_mods/dankeeling/CreativeLiveCameraMount/README.md:1]: Adjusted the link.

File: printer_mods/dankeeling/CreativeLiveCameraMount/STL/Creative_Live_Camera.stl
Comment (DanKeeling) [reply printer_mods/dankeeling/CreativeLiveCameraMount/STL/Creative_Live_Camera.stl:1]: Should only be the one STL now, thanks.

**Review by chirpy2605 — APPROVED** [review]

File: printer_mods/dankeeling/CreativeLiveCameraMount/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+1 x tnut 
+
+Creative Live! Camera - Amazon(https://www.amazon.com/Creative-Wide-Angle-Cancellation-Improved-Universal/dp/B0919XGLXH)
+
+
+Instructions:
+
+printed from ABS 30% infill on its side, supports optional.
+
+Unscrew the one screw that the camera mount hindges on and slide the printed part into its place.
```
Comment (newphreak) [inline printer_mods/dankeeling/CreativeLiveCameraMount/README.md:26]: hindges should probably be hinges? :)

**Review by newphreak — CHANGES_REQUESTED** [review]

File: printer_mods/dankeeling/CreativeLiveCameraMount/README.md (hunk 1 above)
Comment (bryanboettcher) [inline printer_mods/dankeeling/CreativeLiveCameraMount/README.md:26]:
```suggestion
Unscrew the one screw that the camera mount hinges on and slide the printed part into its place. 
```

**Review by bryanboettcher — CHANGES_REQUESTED** [review]

**Review by DanKeeling — COMMENTED** [review]
Spelling change updated

File: printer_mods/dankeeling/CreativeLiveCameraMount/.metadata.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
+---
+# The title of your mod. Choose a short and descriptive name
+title: Creative Live! Camera Mount
+# A short description of your mod. Try to describe your mod with 1-2 short sentences
+description: A mod to provide a mount for the Creative Live! Camera that mounts on 20mm extrusion.
+# Mod version: Set this to 1 when you are submitting a new mod. When you make changes to your mod, you may increment this number
+mod_version: 1
+# You may add your discord username here so people can contact you about your mod
+discord_username: Homelabhobbyist V2.4975
```
Comment (MapleLeafMakers) [inline printer_mods/dankeeling/CreativeLiveCameraMount/.metadata.yml:9]: Ok, I think this is the last thing:  `discord_username` should be the username, not the display name.  I think yours is `klest?`

Comment (bryanboettcher) [reply printer_mods/dankeeling/CreativeLiveCameraMount/.metadata.yml:9]: @MapleLeafMakers I agree with your assessment -- any objection to proposing that as a suggestion so DanKeeling can commit it directly?

**Comment by bryanboettcher** [comment]
@dksub -- thank you for working with us as we work out the kinks in our new PR process.  It is very much appreciated!

File: printer_mods/dankeeling/CreativeLiveCameraMount/.metadata.yml (hunk 2 above)
Comment (DanKeeling) [reply printer_mods/dankeeling/CreativeLiveCameraMount/.metadata.yml]: Made the update!

**Comment by DanKeeling** [comment]
> @DKsub -- thank you for working with us as we work out the kinks in our new PR process. It is very much appreciated!

All good, happy to be a tester and help the community.

**Review by bryanboettcher — APPROVED** [review]
didn't realize I had open requests.

**Comment by bryanboettcher** [comment]
Notice for other reviewers: my approval is not one from an exhaustive PR, I was just closing out my single comment that I somehow turned into a merge-stopping review.

**Pull request merged**